modules/home-manager/home.nix

```
@@ -0,0 +1,30 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  inherit (lib) ctp mkIf;
+  cfg = config.home.catppuccin;
```

```suggestion
  cfg = config.home.pointerCursor.catppuccin;
```
we usually place name our options after the main option we're changing

---

I did try this, but `home.pointerCursor` is a module and I can't define extra options outside of the module

---

- `home.catppuccin.cursor`
	I still think this one is the best
 - `catppuccin.home.pointerCursor`
	 I don't like this one because `catppuccin` is only for global options
 - `home.pointerCursorCatppuccin`
	This one can work too
 - Send PR upstream to not define `home.pointerCursor` as a module
 	Too complex and might not even be possible
 - Move all `*.catppuccin` options to `catppuccin.*` and release new major version, breaking all existing configs
 	Probably too late to change to this approach, but it feels less hacky and more predictable since all options defined by our module are under the same attribute (`catppuccin`). The configuration can be concentrated to the same location in the users' config files. It's also how stylix defines its options.

---

perhaps define it under the `catppuccin.cursor` since `pointerCursor` is a "global" option in the same way... at least thats my thought process

---

i think `catppuccin.pointerCursor` would be a good option for the same reasons isabel mentioned, but also to keep parity with the `home.pointerCursor` option

---

> Move all *.catppuccin options to catppuccin.* and release new major version, breaking all existing configs
Probably too late to change to this approach, but it feels less hacky and more predictable since all options defined by our module are under the same attribute (catppuccin). The configuration can be concentrated to the same location in the users' config files. It's also how stylix defines its options.

https://github.com/catppuccin/nix/issues/222